### PR TITLE
Update customer-key-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
updated unified audit log storage to DEP for multiple Microsoft 365 workloads

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6433

File: microsoft-365/compliance/customer-key-overview.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ A data encryption policy (DEP) defines the encryption hierarchy. This hierarchy
 - Teams status messages
 - User and signal information for Exchange Online
 - Exchange Online mailboxes that aren't already encrypted by mailbox DEPs
+- Unified Audit log storage
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-overview.md]:
```suggestion
- Unified audit log storage
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder

**Pull request merged**